samples/pmic/native/npm2100_one_button/boards/npm2100_pmic.dtsi

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+npm2100_pmic: pmic@74 {
+	compatible = "nordic,npm2100";
+	reg = <0x74>;
+	pmic-int-pin = <1>;
+	pmic-int-flags = <GPIO_ACTIVE_HIGH>;
```

Comment:
Consider adding `host-int-type = "level";`, as this will reduce power consumption at no noticeable delay for the sample, The default is "edge" triggering, which uses GPIOTE events on the nRF5x SoCs